<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calendar</class>
 <widget class="QMainWindow" name="Calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>University Faclilities Management System (UFMS)</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="timedisplay">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>46</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CLOCK</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>431</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0.624591, y1:0.647, x2:0, y2:0, stop:0 rgba(31, 180, 214, 255), stop:1 rgba(183, 183, 183, 255));


</string>
    </property>
    <property name="selectedDate">
     <date>
      <year>2019</year>
      <month>8</month>
      <day>21</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2019</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::ShortDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>14</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Session:</string>
    </property>
   </widget>
   <widget class="QLabel" name="datedisplay">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>46</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>:  DATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="timecurrentLabel">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>46</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Time   :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonmainmenu">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>13</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>MAIN MENU</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dateselectLabel">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>522</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateselectedLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>44</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Today's Date</string>
    </property>
   </widget>
   <widget class="Line" name="frame3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="frame2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="frame1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="frame4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="currentuser_lbl">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>14</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>USERNAME</string>
    </property>
   </widget>
   <widget class="Line" name="frame2_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="frame2_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>40</y>
      <width>20</width>
      <height>481</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="regionlbl">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>San Francisco(US)</string>
     </property>
    </widget>
    <widget class="QLabel" name="flightnumlbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>WW9999</string>
     </property>
    </widget>
    <widget class="QLabel" name="airportlbl">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Kuala Lumpur International Airport (KLIA)</string>
     </property>
    </widget>
    <widget class="QLabel" name="timelbl">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9999</string>
     </property>
    </widget>
    <widget class="QLabel" name="airlinelbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Aeroflot Russian Airlines</string>
     </property>
    </widget>
    <widget class="QLabel" name="flighticon">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>icon</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gatenum">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>99</string>
     </property>
    </widget>
    <widget class="QLabel" name="status">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DELAYED</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnflight">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>125</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SELECT</string>
     </property>
    </widget>
    <widget class="QLabel" name="gatenumlbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GATE        :</string>
     </property>
    </widget>
    <widget class="QLabel" name="regionlblname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ARRIVAL   :</string>
     </property>
    </widget>
    <widget class="QLabel" name="airportlblname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DEPATURE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="timelblname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TIME          :</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusname">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>STATUS:</string>
     </property>
    </widget>
    <widget class="QLabel" name="datetitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DATE         :</string>
     </property>
    </widget>
    <widget class="QLabel" name="datelbl">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>48</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>19999999</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="flightshow">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>46</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Flight(s):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonreturn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>13</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RETURN</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>timedisplay</zorder>
   <zorder>userLabel</zorder>
   <zorder>datedisplay</zorder>
   <zorder>timecurrentLabel</zorder>
   <zorder>buttonmainmenu</zorder>
   <zorder>dateselectLabel</zorder>
   <zorder>dateselectedLabel</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>frame3</zorder>
   <zorder>frame2</zorder>
   <zorder>frame1</zorder>
   <zorder>frame4</zorder>
   <zorder>currentuser_lbl</zorder>
   <zorder>frame2_2</zorder>
   <zorder>frame2_3</zorder>
   <zorder>line</zorder>
   <zorder>groupBox</zorder>
   <zorder>flightshow</zorder>
   <zorder>buttonreturn</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
